recordedfuture.json

```diff
@@ -1,17 +1,17 @@
 {
     "appid": "6efe0e1b-76ac-4ffd-8fa0-ac58fd6efd77",
     "name": "Recorded Future For Splunk SOAR",
-    "description": "This app implements investigative actions to perform lookups for quick reputation information, contextual threat intelligence and external threat alerts",
-    "publisher": "Recorded Future, Inc",
+    "description": "Enhance your security posture with Recorded Future for Splunk SOAR.\nKey Capabilities:\n\u2022Swift Threat Assessments: Access Recorded Future's extensive IOC data for swift and accurate assessments",
+    "publisher": "Splunk",
     "type": "reputation",
     "main_module": "recordedfuture_connector.py",
-    "app_version": "4.3.2",
```

Comment:
@mishalp-crest, the patch version was update by @JQvickWester here. Is there some other file where the version needs to be updated? Thanks.

Reply:
@nplebanskyi Whenever a new action is added to the app, the major version of the app should be updated as per standard practice 
That is the concern

Reply:
I see your point, yes, then it makes sense to update the minor version, thank you.